src/heartbeat/utils.js

```diff
@@ -20,9 +20,12 @@ const queue = require('../utils/queue');
 const httpUtils = require('../utils/httpUtils');
 const errors = require('../errors');
 const collectorStatus = require('../constants').collectorStatus;
-const collect = require('../remoteCollection/collect').collect;
+const collectBulk = require('../remoteCollection/collect').collectBulk;
+const collectNonBulk = require('../remoteCollection/collect').collectNonBulk;
```

Comment:
Let's use better language to describe non-bulk here, like we do in the sgt utils project: if it's non-bulk it is "by subject", e.g. collectBulk, collectBySubject, handleCollectResponseBulk, handleCollectResponseBySubject, etc.

Reply:
good point, easier to understand